index.yaml

```diff
@@ -1,6 +1,19 @@
 apiVersion: v1
 entries:
   sematic-server:
+  - apiVersion: v2
+    appVersion: v0.22.0
+    created: "2023-01-18T01:59:44.866064746Z"
+    description: Sematic AI Server
+    digest: 982e4ef68e13ea72eec1546a2e3ccc526b427a046657e5ddcfe9d4036220a705
+    maintainers:
+    - name: sematic-ai
+      url: https://github.com/sematic-ai/sematic/
+    name: sematic-server
+    type: application
+    urls:
+    - https://sematic-ai.github.io/helm-charts/sematic-server-1.0.1.tgz
```

Comment:
Why is the helm chart version different from the release version?

Reply:
This is the version of the helm charts, which may evolve differently compared to the server version given work being done on the charts themselves.